vendor/vtd-api/Makefile

```diff
@@ -0,0 +1,20 @@
+# Overrides will be preserved in Makefile.package.
+override SOURCE_DIR := src
+override PACKAGE_CHANNEL := cloe/stable
+override CLEAN_SOURCE_DIR := true
+
+include ../../Makefile.package
+
+export:
+	# Only try to export the vtd sources if they are actually there.
+	@( \
+		cat conanfile.py \
+		| sed -rne 's/^.*"(.*\.tgz)"$$/\1/p' \
+		| while read file; do \
+			if [ ! -f $$file ]; then \
+				echo "Missing source tgz: $$file"; \
+				echo "Skipping export of: ${PACKAGE_FQN}"; \
```

Comment:
During `export`, missing source file archives are nicely handled. During `make VENDOR_ONLY=1 package` with the source files not present, I am getting errors in the conan recipe's `build()` method.

Reply:
This should be tested again after PR #5 has been merged. This may resolve the issue.

Reply:
After further investigation, I think the easiest way to go here is to make `WITH_VTD=0` the default. That means that when we want to build the VTD binding, we need to ask for it now with `WITH_VTD=1`.